src/Orleans.Transactions/Abstractions/ITransactionLogStorage.cs

```diff
@@ -8,13 +8,6 @@ namespace Orleans.Transactions.Abstractions
     /// </summary>
     public interface ITransactionLogStorage
     {
-        /// <summary>
-        /// Initialize the log (in Recovery Mode). This method must be called before any other method
-        /// is called on the log.
-        /// </summary>
-        /// <returns></returns>
-        Task Initialize();
```

Comment:
why did we remove the async init method?

Reply:
oh I see, because we are going the factory pattern

Reply:
Yes.  Trying to keep the interface surface geared towards usage pattern, avoiding construction/initialization.